std/process.d

```diff
@@ -570,7 +559,7 @@ private Pid spawnProcessImpl(in char[][] args,
             string errorMsg;
             readExecResult = read(forkPipe[0], &error, error.sizeof);
 
-            switch(status)
+            switch (status)
```

Comment:
I probably misunderstood you first. Do you want assert(0) on noerror case instead of default case?